lib/runner.js

```diff
@@ -54,7 +54,12 @@ class Runner extends Emittery {
 		let hasStarted = false;
 		let scheduledStart = false;
 		const meta = Object.freeze({
-			file: options.file
+			file: this.file,
+			snapshotDirectory: snapshotManager.determineSnapshotDir({
```

Comment:
```suggestion
			get snapshotDirectory: () => snapshotManager.determineSnapshotDir({
```

Reply:
FYI: Apparently you can't use lambdas with the `get` syntax. Must be a regular function with curly brackets, function body and `return` etc..

Reply:
> Apparently you can't use lambdas with the `get` syntax. Must be a regular function with curly brackets, function body and `return` etc..

Why can't we have nice things.